setup-without-nix.sh

```diff
@@ -109,6 +111,19 @@ EOF
       ~ephemeraladmin/.ssh/id_ed25519 \
       ~ephemeraladmin/.ssh/id_ed25519.pub
 
+    mkdir -p /var/lib/buildkite-agent/hooks
+    cat <<EOF > /var/lib/buildkite-agent/hooks/agent-shutdown
```

Comment:
Otherwise `$machine` will get substituted when the script is written, instead of taking the value from the `for` loop.

```suggestion
    cat <<'EOF' > /var/lib/buildkite-agent/hooks/agent-shutdown
```